src/main/java/org/broadinstitute/hellbender/tools/gvs/common/PloidyUtils.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.tools.gvs.common;
+
+import htsjdk.samtools.util.CollectionUtil;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.Lazy;
+import htsjdk.variant.variantcontext.VariantContext;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PloidyUtils {
+    /**
+     * This copies the PAR definition used in the tool FindMendelianViolations with the addition of the Y regions
+     * https://github.com/broadinstitute/picard/blob/master/src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java#L203
+     * Wikipedia data: https://en.wikipedia.org/wiki/Pseudoautosomal_region
+     */
+    private static final Set<String> PSEUDO_AUTOSOMAL_REGIONS_DEFINITION = CollectionUtil .makeSet("X:60001-2699520", "X:154931044-155260560", "chrX:10001-2781479", "chrX:155701383-156030895", "Y:10001-2649520", "Y:59034050-59363566", "chrY:10001-2781479", "chrY:56887903-57217415");
+
+    public static Lazy<Set<Interval>> ParIntervals = new Lazy<>(() -> Collections.unmodifiableSet(parseIntervalLists(PSEUDO_AUTOSOMAL_REGIONS_DEFINITION)));
+
+    public static boolean doesVariantOverlapPAR(VariantContext variant) {
+        return doesIntervalOverlapPAR(variant.getContig(), variant.getStart(), variant.getEnd());
+    }
+
+    public static boolean isLocationInPAR(long location) {
+        // Note: SchemaUtils.decodeContig makes implicit hg38 assumptions. If we support different references, we'll
+        // need to factor that into how we do our mapping from pure location numbers to contigs and positions
```

Comment:
perhaps belongs in method Javadoc rather than an internal comment

Reply:
Hmm, I can totally see that.  I'm not 100% sure, as the comment relates to the particular implementation of the method below (that calls SchemaUtils.decodeContig) more so than the nature of method itself so it _feels_ more attached to that implementation line than a Javadoc about the method, if that makes sense?  I could see a valid case for putting it in either place, but it still feels like more of a reminder that implementing using SchemaUtils internally carries those assumptions with it.

I left this particular one as an inline comment, but added a Javadoc to the class itself that made note of internal hg38 assumptions.